rslib/src/scheduler/states/fuzz.rs

```diff
@@ -34,7 +34,10 @@ static FUZZ_RANGES: [FuzzRange; 3] = [
 impl<'a> StateContext<'a> {
     /// Apply fuzz, respecting the passed bounds.
     pub(crate) fn with_review_fuzz(&self, interval: f32, minimum: u32, maximum: u32) -> u32 {
-        with_review_fuzz(self.fuzz_factor, interval, minimum, maximum)
+        match &self.load_balancer {
+            Some(load_balancer) => load_balancer.find_interval(interval, minimum, maximum),
+            None => with_review_fuzz(self.fuzz_factor, interval, minimum, maximum),
+        }
```

Comment:
> One potential downside that may have been mentioned above (I'll re-read the history tomorow) is that it might lead to predictable spreading of new cards into the available days

Yeah, that seems like the last unresolved issue. New cards can end up being scheduled on the same day. Jake said he'll think about it, but I don't see any way to prevent that.

Reply:
Marking this resolved and bringing it up again below